docs/documentation/powertrain_builder/pt_visual/attributes.rst

```diff
@@ -0,0 +1,67 @@
+.. _attributes:
+
+=========================================
+PT file visualization function attributes
+=========================================
+Other than the key settings—the PT config file path and where you want to save the HTML—you can tweak several other
```

Comment:
```suggestion
Other than the key settings like the PT config file path and where you want to save the HTML, you can tweak several other
```
I'd suggest that because dashes introduce a break in the sentence, whereas here, what you have between the dashes is the logical continuation of the sentence